01 - Community/Obsidian Roundup/2022-06-18 Breaking Changes & Dataview API Upgrades.md

```diff
@@ -0,0 +1,101 @@
+---
+link: https://www.obsidianroundup.org/2022-06-18/
+author: Eleanor Konik
+published: 2022-06-18T12:30:00
+publish: true
+---
+
+# 2022-06-18: Breaking Changes & Dataview API Upgrades
+Multiple windows is now a core feature, allowing for improved dual monitor use of Obsidian. There's also a great new beginner's guide to Dataview.
```

Comment:
Not sure if a missing newline here needs to be flagged but might be related to #535

Reply:
The problem with #535 is the `##` being in the middle of a line.

Blank lines are not needed after (or before) markdown headings.

See https://raw.githubusercontent.com/obsidian-community/obsidian-hub/e6aae4fd147777f0a78c5601fc8263afe262e5a7/01%20-%20Community/Obsidian%20Roundup/2022-03-19%20Better%20Citations%20Workflows%20%26%20Native%20Callout%20Boxes.md

```text
# 2022-03-19: Better Citations Workflows & Native Callout Boxes
The hover editor plugin got secondary support from other plugins, there's a new option for dynamic indexes, & a guide for browser-based Obsidian.
```


Which displays as:
https://publish.obsidian.md/hub/01+-+Community/Obsidian+Roundup/2022-03-19+Better+Citations+Workflows+%26+Native+Callout+Boxes

![image](https://user-images.githubusercontent.com/4840096/202866592-57419c63-e5ce-4c16-a7a4-dce3aab22a4a.png)